osf_pigeon/pigeon.py

```diff
@@ -212,16 +217,46 @@ def get_with_retry(
 async def get_pages(url, page, result={}):
     url = f'{url}?page={page}'
     resp = get_with_retry(url, retry_on=(429,))
-    result[page] = resp.json()['data']
+
+    if 'contributors' in url:
+        result[page] = get_contributors(resp.json())['data']
+    else:
+        result[page] = resp.json()['data']
     return result
 
 
+def get_contributors(response):
+    json_data = response['data']
+    contributor_data_list = []
+    for contributor in json_data:
+        contributor_data = {}
+        embed_data = contributor['embeds']['users']['data']
+        contributor_data['ORCiD'] = embed_data['attributes']['social'].get('orcid', None)
+        contributor_data['name'] = embed_data['attributes']['full_name']
+        links = embed_data['relationships']['institutions']['links']
+        institution_url = links['related']['href']
+        institution_response = get_with_retry(
```

Comment:
We should just change the OSF to make institutions embed-able here, it would save many requests, but obviously a separate ticket.

Reply:
Agreed.

Reply:
Probably not a rush. We'd save, on average, pretty close to one request per registration. Most of the registrations will have 0 institutions, very few will have more than one, almost none will have more than 10, and if we're doing 100 per request, nobody's going to have 100 institutions.